filebeat/module/santa/_meta/fields.yml

```diff
@@ -0,0 +1,42 @@
+- key: santa
+  title: "Google Santa"
+  description: >
+    Santa Module
+  fields:
+    - name: santa
```

Comment:
I assume all fields are on the module level on purpose?

Reply:
Yeah, I'm never expecting there to be anything other than the "log" dataset so dropped the `.log`. Hopefully that's ok.